README.md

```diff
@@ -684,13 +681,17 @@ This library supports dragging within scroll containers (DOM elements that have
 1. The `Droppable` can itself be a scroll container with **no scrollable parents**
 2. The `Droppable` has **one scrollable parent**
 
-**Auto scrolling is not provided (yet!)**
+### Empty `Droppable`s
+
+It is recommended that you put a `min-height` on a vertical `Droppable` or a `min-width` on a horizontal `Droppable`. Otherwise when the `Droppable` is empty there may not be enough of a target for `Draggable` being dragged with touch or mouse inputs to be *over* to `Droppable`.
```

Comment:
> Otherwise when the Droppable is empty there may not be enough of a target for **a** Draggable being dragged with touch or mouse inputs to be over ~~to~~**the** Droppable.

Reply:
nice

Reply:
done 😄